examples/Rules-method-maxDose-IncrementsMaxToxProb.R

```diff
@@ -0,0 +1,17 @@
+model <- LogisticLogNormalOrdinal(
+  mean = c(0.25, 0.15, 0.5),
+  cov = matrix(c(1.5, 0, 0, 0, 2, 0, 0, 0, 1), nrow = 3),
+  ref_dose = 30
+)
+
+emptyData <- DataOrdinal(
+  doseGrid = c(1, 3, 9, 25, 50, 75, 100),
+  yCategories = c("No tox" = 0L, "DLAE" = 1L, "CRS" = 2L)
+)
+
+# For warning regarding tox, see issue #748 https://github.com/openpharma/crmPack/issues/748
```

Comment:
hm, ok to put this here, but then please add to #748 that we need to remove those `suppressWarnings()` so that we don't forget it

Reply:
Done.